### ISSUE TITLE
Pleas add a bit more information about connecting and debugging

### ISSUE THREAD

**Issue by explit28** [iamjackg/esp32-phomemo-gameboy-printer#3]
Could you please add a bit more information or pictures to the Project?

- Its unclear for me how many pins are needed to connect ? Only 3 ?
- What about VCC and Ground Pins on the Gameboy cable?
- Could you attach a pictures which pins of the Gameboy cable are connected?
- How to use `pio run -t monitor`
- When i start it - it can not connect to T02 (Firmware is already installed on the ESP32, T02 is on and work from iPhone and from my Linux PC)

![Screenshot from 2024-12-25 03-10-42](https://github.com/user-attachments/assets/58bb6749-9719-44dd-a202-e833b600e8cf)

**Comment by iamjackg** [comment]
> Its unclear for me how many pins are needed to connect ? Only 3 ?

Yep, you should also connect the GB's ground to the ESP's ground.

> Could you attach a pictures which pins of the Gameboy cable are connected?

I don't have the original thing set up anymore, sorry -- I have since built a shield for the D1 mini and I've been using that to connect to the Game Boy. https://github.com/iamjackg/d1mini-gb-shield

> How to use `pio run -t monitor`

It should be enough to literally just run `pio run -t monitor` in the project folder.

> When i start it - it can not connect to T02

What's the name for the printer that shows up when you pair it? This project is [hardcoded to connect to a device called `T02`](https://github.com/iamjackg/esp32-phomemo-gameboy-printer/blob/acfd698706fe750c6c1046197b5d7f4bde1a3927/src/main.cpp#L93).

**Comment by explit28** [comment]
Thank you, that help me.
Yes the Name is the same like in code.
